api_v3/services/GroupUserService.php

```diff
@@ -434,4 +428,24 @@ protected function validateKuserkGroupCoExistence(kuser $kGroup, $kuserId)
 		
 		throw new KalturaAPIException (KalturaErrors::GROUPS_CANNOT_CO_EXIST, $kuserId, $kGroup->getId(), $kuserKgroupCoExistenceConfig);
 	}
+
+	protected function validateMaxGroupsPerUser($partnerId, $kuserId)
+	{
+		$criteria = new Criteria();
+		$criteria->add(KuserKgroupPeer::KUSER_ID, $kuserId);
+		$criteria->add(KuserKgroupPeer::STATUS, KuserKgroupStatus::ACTIVE);
+
+		$maxGroupsPerUser = GroupUserService::DEFAULT_MAX_NUMBER_OF_GROUPS_PER_USER;
+		$groupUserLimitMap = kConf::get('groupUserLimit');
```

Comment:
if there will be no such param in one of the envs you will get an execption here, make sure you pass local,null in the function params to avoid it

Reply:
I would probably set default value not to null but to an array to have both default and actual value types to be from the same type.

Reply:
The merged version is indeed with a default empty array. 😃